Makefile

```diff
@@ -535,7 +535,10 @@ clean: | clean-common
 	rm -rf bin/* node_modules bottles/* pkg/* tmp/* $(STATUSGO) $(KEYCARDGO)
 	+ $(MAKE) -C vendor/DOtherSide/build --no-print-directory clean
 
-run: $(RUN_TARGET)
+force-rebuild-status-go:
+	bash ./scripts/force-rebuild-status-go.sh
+
+run: force-rebuild-status-go $(RUN_TARGET)
```

Comment:
Another thing, it would be nice if that force-rebuild was also done on `make` and not just `make run`

Reply:
I couldn't find where to plug my task. Do you know where it should be. I am new to this makefile

Reply:
I actually don't know either 🤔 
I think it's the `all` task, being `nim_status_client`